docs/dev/api/accounts.md

```diff
@@ -228,8 +228,6 @@ Creates a new team under the organization of the requesting account.
      <td><p><small>| BODY | REQUIRED | OBJECT |</small></p><p>The settings object specifies the concurrency allocations for the team within the organization. The available attributes are:
      <ul>
       <li><code>virtual_machines</code> - <small>INTEGER</small></li>
-      <li><code>real_devices</code> - <small>INTEGER</small></li>
-      <li><code>live_only</code> - <small>BOOLEAN</small> Defaults to <code>false</code>.</li>
```

Comment:
Could we also remove both `real_devices` and `live_only` from `Sample Response`?

Reply:
The same can be done for `Update a Team`.